website/www/site/assets/scss/_pillars.scss

```diff
@@ -87,17 +96,87 @@
           opacity: 1;
         }
       }
-
-      .pillars-youtube-icon {
-        margin: 0 80px;
-      }
     }
 
     .pillars-social-text {
       @extend .component-text;
       max-width: 285px;
     }
   }
+
+  text-align: center;
```

Comment:
it's unclear where this property belongs. If it belongs to .pillars, move it up:
`.pillars {
  padding: $pad;
  text-align: center;`

Reply:
done